xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -79,16 +79,30 @@
         this.execution = execution;
         this.componentManager = componentManager;
     }
+    
+    /**
+     * Add some content to the mail to be sent. Can be called several times to add different content type to the mail.
+     * This method can be used for scripting languages other than Velocity
+     * @return the Mime Body Part object that was added. Returning it allows script to make modifications to that body
```

Comment:
I don't think the return make much sense since this method always return the passed BodyPart.

Reply:
@tmortagne Just to clarify, do you want to make the method void or modify the description of the return?